### PR TITLE
QuickSort_3_way - Javascript

### PR DESCRIPTION
**Pull request by atrip10**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by jainaman224** [comment]
Please refer to issue number.

File: Comb_Sort/Comb_Sort.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* Function to implement Comb sort using JavaScript
+    
+    Example : If input array is [9,7,8,3,2], then after implementing this code the output array will be [2,3,7,8,9].  
+*/
+
+function combSort(arr){
+      // the shrink factor is taken to be 1.3
```
Comment (jainaman224) [inline Comb_Sort/Comb_Sort.js]: Use 4 space indentation.

File: Comb_Sort/Comb_Sort.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/* Function to implement Comb sort using JavaScript
+    
+    Example : If input array is [9,7,8,3,2], then after implementing this code the output array will be [2,3,7,8,9].  
+*/
+
+function combSort(arr){
```
Comment (jainaman224) [inline Comb_Sort/Comb_Sort.js]: Add space before `{`.

File: Comb_Sort/Comb_Sort.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+/* psuedo data for array */
+var arr = [3,9,8,2,5,1];
+
+/* prints the solution */
+console.log('The sorted array is' + combSort(arr));
+
+/*
+    Output : 1, 2, 3, 5, 8, 9
+*/
```
Comment (jainaman224) [inline Comb_Sort/Comb_Sort.js]: Add a single new line at the end of file.

**Comment by jainaman224** [comment]
Please create separate PR for separate issue. This PR contains files for 3 issues.

**Comment by jainaman224** [comment]
@atrip10 Are you working on it?

**Comment by atrip10** [comment]
Yes, I am working on it

**Comment by atrip10** [comment]
> Please create separate PR for separate issue. This PR contains files for 3 iss

**Comment by atrip10** [comment]
> Please create separate PR for separate issue. This PR contains files for 3 issues.

How to delete other files from this pull request?

**Pull request closed without merging**